specs/signature-specification.md

```diff
@@ -261,9 +261,13 @@ The client implementation can use the aforementioned `mediaType` to parse the si
 
 ## Appendix
 
-### Signing time
+### Signing time & Authentic Signing time
 
-The signing time denotes the time at which the signature was generated. A X509 certificate has a defined [validity](https://datatracker.ietf.org/doc/html/rfc5280#section-4.1.2.5) during which it can be used to generate signatures. The signing time must be greater than or equal to certificate's `notBefore` attribute, and signing time must be less than or equal to certificate's `notAfter` attribute. Signatures generated after the certificate expires are considered invalid. An authentic timestamp, like TSA countersignature, allows a verifier to determine if the signature was generated when the certificate was valid. It also allows a verifier to determine if a signature be treated as valid when a certificate is revoked, if the certificate was revoked after the signature was generated. In the absence of an authentic timestamp, signatures are considered invalid after certificate expires, and all signatures are considered revoked when a certificate is revoked.
+A `signing time` denotes the time at which the signature was generated. An X509 certificate has a defined [validity](https://datatracker.ietf.org/doc/html/rfc5280#section-4.1.2.5) during which it can be used to generate signatures. The signing time must be greater than or equal to certificate's `notBefore` attribute, and the signing time must be less than or equal to certificate's `notAfter` attribute. Signatures generated outside the certificate's validity are considered invalid. 
+
+An `authentic signing time` is a signing time authenticated by a `Signing Authority (SA)`. It allows a verifier to determine if the signature was generated within the certificate's validity. This is ONLY supported under signing scheme [`notary.x509.signingAuthority`](./signing-scheme.md/#notaryx509signingauthority). The `authentic signing time` is determined and MUST be provided by the *Authentic Signing Time* signed attribute in the signature envelope. Signing scheme [`notary.x509`](./signing-scheme.md/#notaryx509) does NOT have the concept of `authentic signing time`.
```

Comment:
Instead of saying `notary.x509` does not have the concept of "authentic signing time", will it be better to say that "authentic signing time" is only available in `notary.x509.signingAuthority` (until we add a new signing scheme to this spec)?